core/src/main/scala/cats/Functor.scala

```diff
@@ -19,22 +19,21 @@ import simulacrum._
 
   def imap[A, B](fa: F[A])(f: A => B, fi: B => A): F[B] = map(fa)(f)
 
-  /////////////////////////////////////////////////////////////////////////
   // derived functions
 
   /**
-    * Lift a function f to operate on Functors
-    */
+   * Lift a function f to operate on Functors
+   */
   def lift[A,B](f: A => B): F[A] => F[B] = map(_)(f)
 
   /**
-    * empty the fa of the values, preserving the structure
-    */
+   * empty the fa of the values, preserving the structure
+   */
   def void[A](fa: F[A]): F[Unit] = map(fa)(_ => ())
 
   /**
-    * Tuple the values in fa with the result of applying a function
-    * with the value */
+   * Tuple the values in fa with the result of applying a function
+   * with the value */
```

Comment:
Someone with a robotic taste for consistency might ask you to move this final `*/` down to the next line to be consistent with the other methods.